adhocracy4/comments/static/comments/CommentBox.jsx

```diff
@@ -51,12 +51,16 @@ class CommentBox extends React.Component {
           })
         }
       })
-      .fail(respone => {
+      .fail(response => {
+        var errorMessage = JSON.parse(response.responseText).comment[0]
```

Comment:
why did you choose `JSON.parse` over the jquery response object `response.responseJSON` [1]. in meinBerlin the later is used [2] but i don't know why

[1] http://api.jquery.com/jquery.ajax/#data-types
[2] f.ex. https://github.com/liqd/a4-meinberlin/blob/310531ffbe8840f280ba7e2daa6b3b7393921192/meinberlin/apps/documents/assets/DocumentManagement.jsx#L272

Reply:
I will check